charts/argo-cd/values.yaml

```diff
@@ -1182,7 +1182,7 @@ redis:
     # -- Redis repository
     repository: public.ecr.aws/docker/library/redis
     # -- Redis tag
-    tag: 7.0.13-alpine
+    tag: 7.0.15-alpine
```

Comment:
Please run `./scripts/helm-docs.sh` and update README
Ref: https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation

Reply:
done

Reply:
Also please bump the image for `redis-ha` component.